tests/dynamic-ui-tool.md

```diff
@@ -0,0 +1,50 @@
+# Dynamic UI tool test cases
+
+This document contains test cases for Dynamic UI tool. The tool depends upon
+DPP Package Manager, so some test cases will depend upon specific tests from
+[the package manager test file](./package-manager.md).
+
+## Premium packages submenu is working
+
+This test verifies whether the premium DPP packages submenu works.
+
+Prerequisites:
+* DTS built with Dynamic UI tool support;
+* `dl.dasharo.com` is up and working properly;
+* Access to a test package with menu script in `dl.dasharo.com`.
+
+Dependencies:
+* Test `A package can be downloaded and installed` from [the package manager
+  test file](./package-manager.md)
+
+Test steps:
+1. Boot into DTS uo to the main menu;
+2. Choose option to provide DPP credentials;
+3. Provide DPP credentials with the access to the test package;
```

Comment:
We have 2 credentials from minio (access key, secret access key). Which should be put where?
 It is very confusing.
 
 ```
 Enter logs key:                1

Enter firmware download key:   2

Enter password:                3
```

Reply:
From the code I understood that for packages use-case:
- logs key - is not used - but I have to provide some random data here
- downlaod key - should be access key (username)
- password - should be secret access key (password)

We will need to document in user-facing documentation with special care.

Reply:
This menu contains three input fields to be able to match with the users that have already bought the subscription and are using it for downloading firmware. You are right that I should have documented that for packages only `download key` and `password` are being used, and we will definitely need to document this.

Reply:
Musimy to zawrzeć w dokumentacji / raporcie dla klienta, aby było wiadomo jak tego użyć, @artur-rs @DaniilKl